src/Layer/C3DTilesLayer.js

```diff
@@ -225,25 +225,6 @@ class C3DTilesLayer extends GeometryLayer {
         }
     }
 
-    /**
-     * Finds the batch table of an object in a 3D Tiles layer. This is
-     * for instance needed when picking because we pick the geometric
-     * object which is not at the same level in the layer structure as
-     * the batch table. More details here on itowns internal
-     * organization of 3DTiles:
-     *  https://github.com/MEPP-team/RICT/blob/master/Doc/iTowns/Doc.md#itowns-internal-organisation-of-3d-tiles-data
-     * @param {THREE.Object3D} object - a 3D geometric object
-     * @returns {C3DTBatchTable} - the batch table of the object
-     */
-    findBatchTable(object) {
```

Comment:
Reason to not expose this method anymore (i.e. remove it)?

Reply:
Mmmh I went a bit too fast I think :grin: However I think it should not have been exposed in the first place (no real use outside of itowns). I can either:
* leave it like this and add a BREAKING CHANGE message
* reintroduce `3DTilesUtils` (with `findBatchTable` in it) and export it (still breaking change but with a fallback elsewhere in the API)
* add it back here and make it call `3DTilesUtils.findBatchTable()`

I'd rather do the first one, but WDYT?

Reply:
I think we'll go for the first option.